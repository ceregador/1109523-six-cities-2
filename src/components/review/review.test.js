import React from 'react';
import Review from './review.jsx';
import Rating from '../rating/rating.jsx';
import Renderer from 'react-test-renderer';

jest.mock(`../rating/rating.jsx`);

it(`renders correctly`, () => {
  const tree = Renderer
      .create(
          <Review
            review={{
              rating: 1,
              user: {
                name: ``,
                avatarUrl: ``
              },
              comment: ``,
              reviewData: ``}}
          />)
      .toJSON();

  expect(Rating).toHaveBeenCalled();
  expect(tree).toMatchSnapshot();
});
