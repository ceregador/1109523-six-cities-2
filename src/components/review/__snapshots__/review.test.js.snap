// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src=""
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <p
      className="reviews__text"
    >
      
    </p>
    <time
      className="reviews__time"
      dateTime="NaN-NaN-NaN"
    >
      Invalid Date
    </time>
  </div>
</li>
`;
